packages/cli/src/controller/init-controller.ts

```diff
@@ -80,7 +80,7 @@ export async function readDefaults(projectPath: string): Promise<string[]> {
 
   const yamlPath = path.join(`${projectPath}`, `project.yaml`);
   const manifest = await fs.promises.readFile(yamlPath, 'utf8');
-  const currentProject = yaml.load(manifest) as ProjectManifestV0_0_1 | ProjectManifestV0_2_0;
+  const currentProject = yaml.load(manifest) as ProjectManifestV0_0_1 | SubstrateProjectManifestV0_2_0;
```

Comment:
`subql init` shouldn't worry about the project's manifest impl. Can you only use the common interface?